service/src/main/resources/db/migration/V415__meal_texture.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE meal_texture
+(
+    id   INTEGER PRIMARY KEY NOT NULL,
+    name TEXT    NOT NULL
+);
+ALTER TABLE child
+    ADD meal_texture_id INTEGER;
+
+ALTER TABLE child
+    ADD CONSTRAINT fk$meal_texture_id_meal_texture_id
+        FOREIGN KEY (meal_texture_id) REFERENCES meal_texture (id)
+            ON DELETE NO ACTION ON UPDATE NO ACTION;
```

Comment:
Should this have `INITIALLY DEFERRED`?

Reply:
No; we use upsert instead of delete all - add all method we discussed earlier